source/wp-content/themes/wporg-news-2021/sass/components/_categories.scss

```diff
@@ -45,21 +45,73 @@ body.category-month-in-wordpress {
 		color: var(--wp--preset--color--off-white-2);
 	}
 
-	.wp-block-post-date a {
-		color: var(--wp--preset--color--off-white-2);
-		text-transform: uppercase;
-	}
-
-	.last-in-year::after {
-		content: "-";
-		display: inline-block;
-		line-height: calc(var(--wp--custom--margin--vertical) * 6);
-		width: 100%;
-		background-image: url(images/brush-stroke-short-blue-4.svg);
-		background-position: center;
-		background-repeat: no-repeat;
-		//background-size: fill;
-		color: var(--wp--preset--color--blue-1);
+	.wp-block-post-template {
+		margin-top: calc(var(--wp--custom--margin--vertical) * 2);
+		margin-bottom: calc(var(--wp--custom--margin--vertical) * 2);
+	}
+
+	.wp-block-post {
+		text-align: center;
+		margin-bottom: calc(var(--wp--custom--margin--vertical) * 1.333);
+
+		@include break-small-only() {
+			width: 50%;
+		}
+	}
+
+	.wp-block-post-date {
+		font-size: var(--wp--preset--font-size--normal);
+
+		a {
+			color: var(--wp--preset--color--off-white-2);
+			text-transform: uppercase;
+		}
+	}
+
+	.last-in-year {
+
+		/* We are removing this item from the grid so we can make the ::after element work as a child of the grid*/
+		display: contents;
+		position: relative;
+		break-after: always;
+
+		&::after {
+			content: "-";
+			line-height: calc(var(--wp--custom--margin--vertical) * 6);
+			width: 100%;
+			mask-image: url(images/brush-stroke-big.svg);
+			mask-position: center;
+			mask-repeat: no-repeat;
+			mask-size: 100% auto;
+			background-color: var(--wp--preset--color--blue-1);
+			transform: translateY(-25%);
+
+			@include break-small() {
+				mask-size: 280px auto;
+			}
+		}
+
+		article {
+
+			/* We are making the article look like it's a child of the grid container */
+			width: calc(33.33333% - 0.83333em);
+			margin-bottom: calc(var(--wp--custom--margin--vertical) * 2.5);
+
+			@include break-small-only() {
+				width: 50%;
+			}
+		}
+	}
+
+	h2 time {
+		font-size: min(max(50px, 5vw), 120px);
+		line-height: 1;
+
+		a:hover,
+		a:focus {
+			color: #7b90ff;
```

Comment:
is there a custom property for this color?

Reply:
I couldn't find it, and the header is also using the hex code too for the active color (it's the same)

Reply:
Aha, I just found it- it's `var(--wp--preset--color--blue-3)`

Reply:
hum, my search didn't find it but you are right!